brave/browser-laptop
# Handle sync network failure errors

Fix https://github.com/brave/browser-laptop/issues/7972

Test plan:
1. turn off wifi
2. try to setup sync in a clean profile. you should see an error message.
3. turn wifi back on.
4. click the 'retry' button in about:preferences#sync
5. sync should now set up


Test Plan:
